conanfile.py

```diff
@@ -142,19 +142,29 @@ def _system_package_architecture(self):
         return ""
 
     def build_requirements(self):
+        pack_names = []
+
         if self.options.GUI:
-            pack_names = []
             if tools.os_info.with_apt:
                 pack_names = ["libxcb1-dev", "libx11-dev", "libc6-dev"]
             elif tools.os_info.is_linux and not tools.os_info.with_pacman:
                 pack_names = ["libxcb-devel", "libX11-devel", "glibc-devel"]
 
-            if pack_names:
-                installer = tools.SystemPackageTool()
-                for item in pack_names:
-                    installer.install(item + self._system_package_architecture())
-
-        if tools.os_info.is_windows and self.settings.compiler == "Visual Studio":
+        if self.options.qtwebengine:
+            # gperf, bison, flex, python >= 2.7.5 & < 3
+            # Should we rely on conan packages for these? I think not, since
+            # these are common build tools that user might have and there's no
+            # need to duplicate them.
+            # https://github.com/bincrafters/conan-winflexbison
+            # https://github.com/conan-community/conan-gperf_installer
+            pack_names += ["bison", "gperf", "flex"]
```

Comment:
Please don't install system package if there is a conan package. You can do the following to avoid duplicating the preinstalled tools:
```
        if not tools.which("bison"):
            self.build_requires("bison_installer/***")
```

Reply:
Cf latest commit. I'm not sure how to find python2 though.